packages/server-wallet/src/config/types.ts

```diff
@@ -84,9 +84,6 @@ export type IncomingServerWalletConfig = RequiredServerWalletConfig &
  * Various network configuration options
  */
 export type NetworkConfiguration = {
-  erc20Address: string;
-  ethAssetHolderAddress: string;
-  erc20AssetHolderAddress: string;
```

Comment:
Keep in mind these values are still being read from the environment

https://github.com/statechannels/statechannels/blob/a67effab94da3301f5dbe21e32e7f8ba8b9afd48/packages/server-wallet/src/chain-service/chain-service.ts#L28-L33

```typescript
const ethAssetHolderAddress = makeAddress(
  process.env.ETH_ASSET_HOLDER_ADDRESS || ethers.constants.AddressZero
);
const nitroAdjudicatorAddress = makeAddress(
  process.env.NITRO_ADJUDICATOR_ADDRESS! || ethers.constants.AddressZero
);
```